production/technical_userguide.md

```diff
@@ -67,15 +65,42 @@ Client Secret:
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-In the following cURL command, notice that we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which performs Basic authentication by passing in an additional header**
+
+Format:
 
 ```
-curl -d '' -X POST "https://api.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic MDk4NjlhN2YtNDZjZS00OTA4LWE5MTQtNjEyOWQwODBhMmFlOjY0OTE2ZmU5NmY3MWFkYzc5YzU3MzVlNDlmNGU3MmYxOGZmOTQxZDBkZDYyY2Y0M2VlMWFlMDg1N2UyMDRmMTczYmExMGU0MjUwYzEyYzQ4"
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
 ```
 
-**Response**
+Example:
+
+```
+curl -d '' -X POST "https://sandbox.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOm\
+Y5NzgwZDMyMzU4OGYxY2RmYzNlNjNlOTVhOGNiZGNkZDQ3NjAy\
+ZmY0OGE1MzdiNTFkYzVkNzgzNGJmNDY2NDE2YTcxNmJkNDUwOGU5MDRh"
+```
+
+**Option 2, which does only requires you to submit your clientId and secret**
+
 
-You will receive a `200 OK` response and an access token if your credentials were accepted, with a body like so:
+Format:
```

Comment:
Is the format section necessary, since we have the example below?

Reply:
Keeping open; related to conversation in https://github.com/CMSgov/bcda-static-site/pull/57#discussion_r415877240

Reply:
To me, the `Format` section seems a bit confusing, because it shows an incomplete `curl` command.  

What if, instead of showing `curl` in the `Format` section, we simply describe what should be done for Basic Auth as part of this option?  That way, the incomplete `curl` in the `Format` section doesn't confuse folks looking at the real `curl` in the `Example` section.

So, for this one (option 2) we could say something like (please make this sound better:
```
With this option, the `clientId:secret` can be passed to the `--user` flag of the `curl` command.  By doing this, `curl` will base64 encode the credentials and pass it to the desired endpoint as an HTTP header.
```

Reply:
Accepted, resolved.